packages/form-field/src/FormFieldFeedback/FormFieldFeedback.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { color } from '@leafygreen-ui/tokens';
+import { Body, Error, useUpdatedBaseFontSize } from '@leafygreen-ui/typography';
+
+import { LGIDS_FORM_FIELD } from '../constants';
+import { getFontSize } from '../FormField/FormField.styles';
+import { FormFieldState } from '../FormField/FormField.types';
+
+import {
+  containerStyles,
+  getIconWrapperHeight,
+  hideContainerStyle,
+  iconWrapperStyles,
+  spacingTop,
+} from './FormFieldFeedback.styles';
+import { FormFieldFeedbackProps } from './FormFieldFeedback.types';
+
+export const FormFieldFeedback = ({
+  disabled,
+  errorMessage,
+  hideFeedback = false,
+  id,
+  size,
+  state,
+  successMessage,
+  ...rest
+}: FormFieldFeedbackProps) => {
+  const { theme } = useDarkMode();
+  const baseFontSize = useUpdatedBaseFontSize();
+  const fontStyles = getFontSize({ baseFontSize, size });
+
+  const isErrorState = state === FormFieldState.Error;
+  const isValidState = state === FormFieldState.Valid;
+  const showFormFieldFeedback = (isErrorState || isValidState) && !disabled;
+
+  const iconProps = {
+    glyph: isErrorState ? 'Warning' : 'Checkmark',
+    fill: color[theme].icon[isErrorState ? 'error' : 'success'].default,
+    title: isErrorState ? 'Error' : 'Valid',
+  } as const;
+
+  return (
+    <div
+      id={id}
+      data-testid="lg-form_field-feedback"
+      className={cx(containerStyles, {
+        [spacingTop]: showFormFieldFeedback,
+        [hideContainerStyle]: hideFeedback,
+      })}
+      aria-live="polite"
+      aria-relevant="all"
+      {...rest}
+    >
+      {showFormFieldFeedback && (
+        <>
+          <div className={cx(iconWrapperStyles, getIconWrapperHeight(size))}>
+            <Icon {...iconProps} aria-hidden />
+          </div>
+          {isErrorState ? (
+            <Error
+              data-lgid={LGIDS_FORM_FIELD.errorMessage}
+              data-testid="lg-form_field-feedback-error_message"
```

Comment:
`data-lgid` and `data-testid` should be the same. I missed updating`data-testid` to be `LGIDS_FORM_FIELD.errorMessage` while i was working on test-harnesses. The difference between the two is that data-testid was there first and is probably used by consumers and `data-lgid` is used inside our test-harnesses. If we're sticking with exporting our test ids, we should update `LGIDS_FORM_FIELD.errorMessage` to be `lg-form_field-feedback-error_message`.

Reply:
thanks! this is helpful context. I will keep them consistent with what was previously being used since consumers may be relying on that in their tests